scan/lib/scan/options.rb

```diff
@@ -97,6 +97,12 @@ def self.available_options
                                      env_name: "SCAN_BUILDLOG_PATH",
                                      description: "The directory were to store the raw log",
                                      default_value: "#{FastlaneCore::Helper.buildlog_path}/scan"),
+        FastlaneCore::ConfigItem.new(key: :include_simulator_logs,
+                                     env_name: "SCAN_INCLUDE_SIMULATOR_LOGS",
+                                     description: "Should the logs of the app on the Simulator be written to the output_directory",
```

Comment:
Sounds good, do you mind clarifying the option's description to mention "the app's output (e.g. NSLog)" or similar 👍